apis/identity/v1beta1/iamrole_types.go

```diff
@@ -38,13 +34,13 @@ type IAMRoleParameters struct {
 	Description string `json:"description,omitempty"`
```

Comment:
```suggestion
	Description *string `json:"description,omitempty"`
```

https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#pointer-types-and-markers